prow/images/prow-tools/README.md

```diff
@@ -0,0 +1,17 @@
+# Prow Tools
+
+## Overview
+
+The directory contains Dockerfile for prow tools image with prebuilt tools used in prow pipelines.
+The image is used as a standalone image in several cleaners job and as additional dependency in `kyma-integration` images.
```

Comment:
```suggestion
The image is used as a standalone image in several cleaners jobs and as an additional dependency in the `kyma-integration` images.
```

I'm guessing it's `jobs` since there are `several`...? Correct me if I'm reading it wrong.